### ISSUE TITLE
should it count paid books in the list or only free books?

### ISSUE THREAD

**Issue by avelino** [avelino/awesome-go#3956]
A discussion arose within a pull request #3952 where I (Avelino) was not accepting paid books - without having the knowledge that there were already paid books on the list - sorry.

- comment on your opinion about having paid books

**Comment by avelino** [comment]
I start by apologizing for not knowing that we had paid books on awesome-go, for me there were only free books

I think it is important to have a subcategory within "books" for non-free books (ebooks can be a subcategory of books) - I am about to restructure it

I am thinking of the following structure:
```md
## books

> here contains paid books

- abc...
- abc...

### free books

- abc...
- abc...
```

what is your opinion? @bitfield can you help us with the book category?

**Comment by bitfield** [comment]
Sounds good to me! How about 'free books' versus 'books for purchase'?

**Comment by avelino** [comment]
> Sounds good to me! How about 'free books' versus 'books for purchase'?

I like the proposal of the names
Thinking about structure (groups and subgroups) I think as follows:

- books - Does not contain even one item, serving to group the subcategories together
  - free books
  - Books for purchase

**Comment by bitfield** [comment]
Pending a final decision on this, would it be all right for me to add my "book for purchase", per #3908? Writing Go books is literally my livelihood, and a listing here would make a big difference to me, as a one-person writing and publishing operation.

It's not just about me, either: what I make from selling ebooks covers the bills, and essentially funds me to provide free Go learning resources such as tutorials and YouTube videos.

If that's not allowed, I quite understand. But then it's hard to square that with the current inclusion of several paid-for books published by Manning, for example (see #3952). They are a large commercial publishing firm who must make many hundreds of dollars a year from people purchasing through the links here. They could probably scrape by without that revenue: I can't.

**Comment by phanirithvij** [comment]
@bitfield I am in favor of this, I believe Avelino is too, from his previous comment. I think you may go ahead and send a PR that restructures these books and following that, another PR that adds your book to the `books for purchase` section.
